src/docs/drizzle/.pages

```diff
@@ -0,0 +1,6 @@
+nav:
+  - Overview: index.md
+  - quickstart.md
+  - getting-started
+  - react
+  - reference
```

Comment:
Needing a new line at the end of the file, same for below :-)

Reply:
Fixed!